src/main/SegCtlr.Netstandard/Control/SegmentedControl.cs

```diff
@@ -89,7 +89,16 @@ public Color DisabledColor
         }
 
 
-        public static readonly BindableProperty SelectedSegmentProperty = BindableProperty.Create(nameof(SelectedSegment), typeof(int), typeof(SegmentedControl), 0);
+        public static readonly BindableProperty SelectedSegmentProperty = BindableProperty.Create(nameof(SelectedSegment), typeof(int), typeof(SegmentedControl), 0, propertyChanged:OnSelectedSegmentChanged);
```

Comment:
```suggestion
        public static readonly BindableProperty SelectedSegmentProperty = BindableProperty.Create(nameof(SelectedSegment), typeof(int), typeof(SegmentedControl), 0, propertyChanged:(bindable, oldValue, newValue) =>
{
    var segmentedControl = (SegmentedControl)bindable;
    segmentedControl.RaiseSelectionChanged();
});
```

Reply:
You could write the propertyChanged directly on the bindable property.

Reply:
I was following the coding style of the file, the `propertychanged` for  `ItemsSourceProperty` is written in the same fashion.

Reply:
I like it. Thank you @FabriBertani